### PR TITLE
[Docs] Update outdated info on website FAQ

### PR DESCRIPTION
**Pull request by jonwzheng**
Related to larger issue of outdated docs in #2703.

This PR serves to eliminate outdated information in the RMG GitHub page, primarily the FAQ although other sections of the website may be updated as well.

### CODE REVIEW

File: documentation/source/users/arkane/credits.rst
Comment (JacksonBurns) [inline documentation/source/users/arkane/credits.rst:1]: While we are editing this file, could you change the "How to Cite" section to just point to the [CITATIONS.bib](https://github.com/ReactionMechanismGenerator/RMG-Py/blob/main/CITATIONS.bib) file we keep in the main repo? That is the most up to date, and I think it would be better to just have that be the singular reliable source for citation instructions.

File: documentation/source/users/rmg/credits.rst
Comment (JacksonBurns) [inline documentation/source/users/rmg/credits.rst:1]: Same comment as on the other credits file, please point to the citation file.

File: documentation/source/users/rmg/database/kinetics.rst [hunk 1]
```diff
@@ -52,150 +52,7 @@ Kinetic libraries should also be used in the cases where:
...
-|naphthalene_H                                                |Cyclopentadiene pyrolysis in the presence of ethene Part 1                                |
-+-------------------------------------------------------------+------------------------------------------------------------------------------------------+
-|vinylCPD_H                                                   |Cyclopentadiene pyrolysis in the presence of ethene Part 2                                |
-+-------------------------------------------------------------+------------------------------------------------------------------------------------------+
-|PrimaryH2O2                                                  |Updated rate parameters for the H2O2 system that include ter-molecular reactions          |
-+-------------------------------------------------------------+------------------------------------------------------------------------------------------+
+A full list of the libraries `can be found on the RMG website: <https://rmg.mit.edu/database/kinetics/libraries/>`_.
```
Comment (JacksonBurns) [inline documentation/source/users/rmg/database/kinetics.rst:55]: Nice

File: documentation/source/users/rmg/database/kinetics.rst [hunk 2]
```diff
@@ -212,237 +69,7 @@ Each reaction family contains the files:
...
-    **intra_NO2_ONO_conversion**                          |f69|
-    **intra_OH_migration**                                |f70|
-    **intra_substitutionCS_cyclization**                  |f71|
-    **intra_substitutionCS_isomerization**                |f72|
-    **intra_substitutionS_cyclization**                   |f73|
-    **intra_substitutionS_isomerization**                 |f74|
-    **lone_electron_pair_bond**                           |f76|
-    ===================================================== =====
-
+A full list of the kinetic families `can be found on the RMG website: <https://rmg.mit.edu/database/kinetics/families/>`_. .
```
Comment (JacksonBurns) [inline documentation/source/users/rmg/database/kinetics.rst:72]: My only note here is that this page takes a long time to load (though it contains more info, so it is worth it). Maybe we try to speed up that page loading (cache it?), and/or put a warning here about long load times?

File: documentation/source/users/rmg/faq.rst [hunk 3]
```diff
@@ -9,44 +9,19 @@ We have compiled some common questions about installing and using RMG below.
...
-   The benefit of this option is being able to run in a full Linux environment. However, running two operating systems
-   simultaneously does result in excess resource overhead, so it may not be suitable for running extended RMG jobs.
-   Instructions for setting up a virtual machine can be found :ref:`here<virtualMachineSetup>`.
-
-   A third option that we are currently beginning to explore using `Docker <https://www.docker.com/>`_, which is a
-   container-based infrastructure which shares the same benefits as a virtual machine but with less overhead. There
-   are some test images of RMG-Py which can be found on `Docker Hub <https://hub.docker.com/>`_ if you would like to
-   give this a try. More detailed instructions will be made available once we officially support this approach.
-
 #. **Windows binary installation gives ``WindowsError: [Error 5]``?**
```
Comment (JacksonBurns) [inline documentation/source/users/rmg/faq.rst:25]: I would remove this as well - not relevant anymore.

File: documentation/source/users/rmg/faq.rst
Comment (JacksonBurns) [inline documentation/source/users/rmg/faq.rst:1]: Please also remove these irrelvant lines: https://github.com/ReactionMechanismGenerator/RMG-Py/blob/5ba15bab143ee88513c675b158912e54610b8592/documentation/source/users/rmg/faq.rst?plain=1#L76-L92

The first is only caused when manually compiling dependencies (our users don't do that anymore) and the second is a Windows problem when running from source (no longer possible).

File: documentation/source/users/rmg/installation/dependencies.rst [hunk 4]
```diff
@@ -8,49 +8,9 @@ Dependencies
 List of Dependencies
 ====================
 
-Briefly, RMG depends on the following packages, almost all of which can be found in the `RMG anaconda channel <https://anaconda.org/rmg>`_ as binary packages.
+A list of RMG's dependencies can be found in the ``environment.yml`` file on the `RMG GitHub page: <https://github.com/ReactionMechanismGenerator/RMG-Py/blob/main/environment.yml>`_.
```
Comment (JacksonBurns) [inline documentation/source/users/rmg/installation/dependencies.rst:11]: Nice

File: documentation/source/users/rmg/installation/dependencies.rst [hunk 5]
```diff
@@ -59,8 +19,3 @@ License Restrictions on Dependencies
 All of RMG's dependencies except the ones listed below are freely available and compatible with RMG's open source MIT license (though the specific nature of their licenses vary). 
 
 * **pydas**: The DAE solvers used in the simulations come from `Linda Petzold's research group <https://cse.cs.ucsb.edu/software/>`_ at UCSB.  For running sensitivity analysis in RMG, the DASPK 3.1 solver is required, which "is subject to copyright restrictions” for non-academic use. Please visit their website for more details. To run RMG without this restriction, one may switch to compiling with the DASSL solver instead in RMG, which is "available in the public domain.”
-
-If you wish to do on-the-fly quantum chemistry calculations of thermochemistry (advisable for fused cyclic species in particular, where the ring corrections to group additive estimates are lacking),
-the then you will need the third-party software for the QM calculations:
-
-* **gaussian**: Gaussian03 and Gaussian09 are currently supported and commercially available.  See `https://gaussian.com <https://gaussian.com>`_ for more details.
```
Comment (JacksonBurns) [inline documentation/source/users/rmg/installation/dependencies.rst:66]: Why was this removed?

**Review by JacksonBurns — CHANGES_REQUESTED** [review]
Thanks for getting this underway! Some notes

File: documentation/source/users/rmg/installation/dependencies.rst (hunk 5 above)
Comment (jonwzheng) [reply documentation/source/users/rmg/installation/dependencies.rst]: My initial thoughts were that this is out of date and needs to be updated. I also had the impression people aren't actively using it / it may be broken (see https://github.com/ReactionMechanismGenerator/RMG-Py/issues/1864), so to be honest I'm not sure what the current best practices for the QMTP module are. That said, on second thought we should definitely mention/include it here. latest changes include an attempt at revising this

Comment (jonwzheng) [reply documentation/source/users/rmg/installation/dependencies.rst]: From the PR above, it looks like it's not quite as necessary for fused rings, so I suggest softening the language

**Comment by jonwzheng** [comment]
Thanks Jackson, I made the changes you suggested above.

File: documentation/source/users/rmg/database/kinetics.rst (hunk 2 above)
Comment (JacksonBurns) [reply documentation/source/users/rmg/database/kinetics.rst]: Great! I think that is an appropriate solution for this PR.

**Review by JacksonBurns — APPROVED** [review]
I think this is ready.

**Comment by JacksonBurns** [comment]
@rwest, @jonwzheng and I are doing an overhaul of the docs as discussed at our board meeting. #2703 will eventually be what goes into main, but we are separating updates to the FAQ, Wiki, and Contributor guidelines into 3 sub-PRs for easier reviewing. Please review this PR, which updates the FAQ (and some other lint in the docs).

File: documentation/source/users/arkane/credits.rst [hunk 6]
```diff
@@ -41,11 +41,4 @@ Developers: (rmg_dev@mit.edu)
...
 
-A. Grinberg Dana, M.S. Johnson, J.W. Allen, S. Sharma, S. Raman, M. Liu, C.W. Gao, C.A. Grambow, M.J. Goldman,
-D.S. Ranasinghe, R.J. Gillis, A.M. Payne, Y.-P. Li, X. Dong, K.A. Spiekermann, H. Wu, E.E. Dames, Z.J. Buras,
-N.M. Vandewiele, N.W. Yee, S.S. Merchant, B. Buesser, C.A. Class, C.F. Goldsmith, R.H. West, W.H. Green,
-"Automated reaction kinetics and network exploration (Arkane):
-A statistical mechanics, thermodynamics, transition state theory, and master equation software",
-*International Journal of Chemical Kinetics* 2023, 55(6), 300-323.
-
-DOI: `10.1002/kin.21637 <https://doi.org/10.1002/kin.21637>`_
+Please refer to the `CITATIONS.bib file <https://github.com/ReactionMechanismGenerator/RMG-Py/blob/main/CITATIONS.bib>`_.
```
Comment (rwest) [inline documentation/source/users/arkane/credits.rst:44]: Could maybe add to that file comments about when to cite what. But maybe that's not helpful, or already obvious.

File: documentation/source/users/arkane/credits.rst [hunk 7]
```diff
@@ -25,8 +25,8 @@ Developers: (rmg_dev@mit.edu)
 - Dr. R.J. Gillis
 - Dr. A.M. Payne
 - Asst. Prof. Y.-P. Li
```
Comment (rwest) [inline documentation/source/users/arkane/credits.rst:27]:
```suggestion
- Prof. Y.-P. Li
```

Comment (rwest) [reply documentation/source/users/arkane/credits.rst:27]: I suggest remove the Asst from Alon above too but GitHub app won't let me comment on that line

File: documentation/source/users/rmg/credits.rst [hunk 8]
```diff
@@ -16,16 +16,14 @@ Project Supervisors:
...
-- Hao-Wei Pang
+- Dr. Anna Doner
 - Oscar Wu
-- Kevin Spiekermann
 - Jonathan Zheng
 - Jackson Burns
 - Nathan Morgan
+- Prof. Richard H. West
 - Prof. C. Franklin Goldsmith
 - Dr. Katrin Blondal
```
Comment (rwest) [inline documentation/source/users/rmg/credits.rst:28]:
```suggestion
```

File: documentation/source/users/rmg/credits.rst [hunk 9]
```diff
@@ -35,6 +33,10 @@ Current Developers: (rmg_dev@mit.edu)
 
 Previous Developers: 
 
+- Dr. Xiaorui Dong
+- Dr. Hao-Wei Pang
+- Dr. Kevin Spiekermann
+- Yen-Ting Wang
 - Dr. Joshua W. Allen
 - Dr. Yunsie Chung
```
Comment (rwest) [inline documentation/source/users/rmg/credits.rst:41]:
```suggestion
- Dr. Katrin Blondal
- Dr. Yunsie Chung
```

File: documentation/source/users/rmg/credits.rst [hunk 10]
```diff
@@ -16,16 +16,14 @@ Project Supervisors:
...
+- Dr. Anna Doner
 - Oscar Wu
-- Kevin Spiekermann
 - Jonathan Zheng
 - Jackson Burns
 - Nathan Morgan
+- Prof. Richard H. West
 - Prof. C. Franklin Goldsmith
 - Dr. Katrin Blondal
 - Dr. Bjarne Kreitz
```
Comment (rwest) [inline documentation/source/users/rmg/credits.rst:29]:
```suggestion
- Dr. Bjarne Kreitz
- Chao Xu
```

**Review by rwest — COMMENTED** [review]
Looks good. Apologies if I messed up formatting my suggestions - did it from my phone.

File: documentation/source/users/arkane/credits.rst (hunk 6 above)
Comment (jonwzheng) [reply documentation/source/users/arkane/credits.rst]: added a brief mention of which to use

**Comment by jonwzheng** [comment]
Thanks for the review @rwest, I included your suggestions in the latest commits.

**Review by rwest — APPROVED** [review]
